workspace/src/app/views/shared/RuleEditor/view/sidebar/RuleOperatorList.tsx

```diff
@@ -0,0 +1,91 @@
+import { Button, Card, List, OverviewItem, OverviewItemActions, Spacing } from "gui-elements";
+import { extractSearchWords } from "gui-elements/src/components/Typography/Highlighter";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { IRuleOperator } from "../../RuleEditor.typings";
+import { RuleOperator } from "./RuleOperator";
+
+interface RuleOperatorListProps {
+    /** The rule operators that should be shown. */
+    ruleOperatorList: IRuleOperator[];
+    /** The text search query. */
+    textQuery: string;
+}
+
+/** The list of rule operators that is shown in the sidebar of the rule editor. */
+export const RuleOperatorList = ({ ruleOperatorList, textQuery }: RuleOperatorListProps) => {
+    const { t } = useTranslation();
+    const searchWords = extractSearchWords(textQuery, true);
+    const [currentlyCycledTaskId, setCurrentlyCycledTaskId] = React.useState<string | undefined>(undefined);
+    const [taskCycleIndex, setTaskCycleIndex] = React.useState<number>(0);
+    const totalMatches = 0; // TODO
+
+    const resetCycleTask = () => {
+        // TODO
+    };
+    const cycleThroughTaskNodes = (operatorId: string) => {
+        // TODO
+    };
+
+    /* Add operator plugin data to drag event. */
+    const onDragStartByPluginId = (pluginType: string, pluginId: string) => (e: React.DragEvent<HTMLDivElement>) => {
+        const pluginData = JSON.stringify({ pluginType, pluginId });
+        e.dataTransfer.setData("text/plain", pluginData);
+    };
+
+    const itemRenderer = (ruleOperator: IRuleOperator) => {
+        /** currently active taskItem */
+        const isActiveTaskItem = currentlyCycledTaskId === ruleOperator.pluginId;
+        return (
+            <div
+                data-test-id={"ruleEditor-sidebar-draggable-operator"}
+                key={ruleOperator.pluginId}
+                draggable={true}
+                onDragStart={onDragStartByPluginId(ruleOperator.pluginType, ruleOperator.pluginId)}
+            >
+                <Card data-test-id={"ruleEditor-sidebar-draggable-operator-" + ruleOperator.pluginId} isOnlyLayout>
```

Comment:
Is there a reason why this markup is in the list but it looks like it should be part of the rule operator component?